bubble-wrap-style.yaml

```diff
@@ -2239,19 +2307,12 @@ layers:
                         - [13, 1.50]
                         - [14, 2.0]
                     cull_from_tile: true
-                    visible: false
+                    visible: true
```

Comment:
Turning this on by default in the past has caused _too_ many shields to show up. We've cleaned up shields server side quite a bit since then, so worth reevaluating. But this may cause visual problems.